review/fileToReview.js

```diff
@@ -1,73 +1,90 @@
+"use strict";
+const axios = require("axios");
 
-var domain = "bank.local.fr"
+const domain = "bank.local.fr";
 
 /**
  * @description Fetch transactions recursively
  * @param {string} fromDate The maximum date of transactions to return
- * @param {string} authorization Authorization header to authent the user
+ * @param {string} authorization Authorization header to authenticate the user
  * @param {jws} jws Jws token, mandatory if we get one from login request
  * @param {Number} id Account id
- * @param {Number} page Number of the page
- * @param {Object} previousTransactions Previous page of transactions (To ckeck for dupes)
+ * @param {Number} page Current page number
+ * @param {Object} previousTransactions Previous page of transactions (To check for duplicates)
  * @return {Object} All transactions available on the page
  */
-async function fetchTransactions(fromDate, authorization, jws = null, id, page, previousTransactions) {
-	console.log(`--- Fetch Trasactions page n°${page} ---`);
-	try {
-    var headers = {"Authorisation":  authorization }
```

Comment:
change this to let and change the object if we really need it (only if we have a jws)